### PR TITLE
feat: added Home Page

### PR DESCRIPTION
**Pull request by srv-pranjal**
# Home

## Description

 - added Header
 - added Categories section
 - added categories database
 
 
image

### CODE REVIEW

**Review by TejasShekar — APPROVED** [review]
Clean code
Good folder structure
Looks good to merge

File: src/components/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export { Navbar } from "./Navbar/Navbar";
```
Comment (sushil-mod) [inline src/components/index.js:1]: I like how you doing barrel export

File: src/pages/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Categories, CategoryCard } from "components";
+import "./Home.css";
+import { categories } from "data/categories";
+
+export const Home = () => {
+  console.log(categories);
```
Comment (sushil-mod) [inline src/pages/Home/Home.js]: remove this log if not in future use

**Review by sushil-mod — APPROVED** [review]
Good start to the project. I like the UI and hovering effect on categories. just a log there rest code is good to go

File: src/pages/Home/Home.js (hunk 2 above)
Comment (srv-pranjal) [reply src/pages/Home/Home.js]: Done, Thanks

**Pull request merged**